observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,99 @@ You can use the {TempoName} xref:../../observability/otel/otel-forwarding-teleme
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="distr-tracing_3-5_{context}"]
+== Release notes for {DTProductName} 3.5
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+[id="distr-tracing_3-5_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* ???
+
+[id="distr-tracing_3-5_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} 3.5 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.??.?].
+
+[NOTE]
+====
+The {TempoName} 3.5 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.?.?.
+====
+
+[id="distr-tracing_3-5_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* With this update, the `traces.span.metrics` namespace becomes the default metrics namespace on which the Jaeger query retrieves the Prometheus metrics. The purpose of this change is to provide compatibility with the OpenTelemetry Collector version 0.109.0 and later where this namespace is the default. Customers who are still using an earlier OpenTelemetry Collector version can configure this namespace by adding the following field and value: `spec.template.queryFrontend.jaegerQuery.monitorTab.redMetricsNamespace: ""`.
```

Comment:
@rubenvp8510, can please check this to see if this is correct? If not, please provide a suggestion to correct it.

Reply:
This is correct!